pkgs/applications/audio/sidplayfp/default.nix

```
@@ -1,20 +1,32 @@
-{ stdenv, fetchurl, pkgconfig, libsidplayfp }:
+{ stdenv, lib, fetchurl
+, pkgconfig
+, libsidplayfp
+, alsaSupport ? stdenv.isLinux, alsaLib
+, pulseSupport ? stdenv.isLinux, libpulseaudio
+}:
 
-stdenv.mkDerivation rec {
-  version = "1.4.4";
+let
+
+  opt = lib.optional;
```

i would like to keep this as consistent as possible with other derivations
```suggestion
  optional = lib.optional;
```

---

I'll replace this and all the `lib.`s in the library with `inherit (lib)`s.